docs/source/faq.rst

```diff
@@ -494,3 +494,49 @@ There are two kinds of constrained optimizations, one with soft constraints and
 Soft constraints do not have to be satisfied, but an objective function is penalized if they are unsatisfied. On the other hand, hard constraints must be satisfied.
 
 Optuna is adopting the soft one and **DOES NOT** support the hard one. In other words, Optuna **DOES NOT** have built-in samplers for the hard constraints.
+
+
+Can I monitor trials and make them failed automatically when they are killed by a scheduler?
+--------------------------------------------------------------------------------------------
+
+.. note::
+
+  Heartbeat mechanism is experimental. API would change in the future.
+
+Optuna supports monitoring trial heartbeats with RDB and Redis storages.
```

Comment:
How about referring to [the Wikipedia page](https://en.wikipedia.org/wiki/Heartbeat_(computing)) or briefly explaining what is the heartbeat.

Reply:
da2454a adds the wikipedia link and improves the explanation. Please give me a feedback/